app/controllers/survey_controller.rb

```diff
@@ -2,4 +2,31 @@ class SurveyController < ApplicationController
   def index
     @questions = Question.order(:order)
   end
+  
+  def createQuestion
+    @question = Question.create(question: params[:question])
+    @question.order = @question.id # Did this because it is currently the convention in our seed data.
+    @question.save!
+    nextOrder = 0
```

Comment:
Doesn't seem like we need to initialize it at zero? you can just do order: 1, order: 2, etc. Why does it need to be dynamic?